tests/testthat/test-all.R

```diff
@@ -419,6 +419,17 @@ test_that("incorrect API Query Human Readable", {
   expect_equal(9, ncol(df), label="columns") 
 })
 
+context("URL suffixes from Socrata are handled")
+
+test_that("Handle /data suffix", {
```

Comment:
Since we're only testing the ability to correctly parse URLs that end in `data` or `data/` I would make the check just check that rather than checking the entire download process.

Reply:
I'm inclined to agree. Can be a later touch-up.

/cc @nicklucius